server/main.lua

```diff
@@ -202,7 +210,23 @@ lib.callback.register('qbx_garages:server:parkVehicle', function(source, netId,
     assert(Garages[garage] ~= nil, string.format('Garage %s not found. Did you register this garage?', garage))
     local vehicle = NetworkGetEntityFromNetworkId(netId)
     local vehicleId = Entity(vehicle).state.vehicleid or exports.qbx_vehicles:GetVehicleIdByPlate(GetVehicleNumberPlateText(vehicle))
-    local owned = isParkable(source, vehicleId, garage) --Check ownership
+    local owned = isParkable(source, vehicleId, garage)
+
+    local payloadPark = {
+        source = source,
+        netId = netId,
+        props = props,
+        garageName = garage,
+        vehicleId = vehicleId,
+        owned = owned,
+        vehicle = vehicle,
+    }
+
+    if GaragesHooks('parkVehicle', payloadPark) == false then
+        lib.print.debug("Vehicle park was canceled by a hook.")
+        return
+    end
```

Comment:
```suggestion
    if not GaragesHooks('parkVehicle', payloadPark) then
        lib.print.debug("Vehicle park was canceled by a hook.")
        return
    end
```
Should we include a notification to the player? Otherwise they may be confused why the vehicle is not spawning.

Reply:
the notification should be added by the code that refuses to park the vehicle